cabal.project

```diff
@@ -4,10 +4,5 @@ jobs: $ncpus
 
 source-repository-package
   type: git
-  location: https://github.com/joshvera/proto3-suite.git
-  tag: 83f3352f0c7c94ea091e6087f60692eda9991fae
-
-source-repository-package
-  type: git
-  location: https://github.com/joshvera/proto3-wire.git
-  tag: 84664e22f01beb67870368f1f88ada5d0ad01f56
+  location: https://github.com/tclem/proto-lens-jsonpb
+  tag: e4d10b77f57ee25beb759a33e63e2061420d3dc2
```

Comment:
Is this something we can/should push to hackage at some point?

Reply:
We definitely can, I'm not sure we should just yet. Let's see if this approach works for us.